pkg/cluster/snow.go

```diff
@@ -268,3 +271,21 @@ func ValidateSnowMachineRefExists(c *Config) error {
 	}
 	return nil
 }
+
+func validateSnowUnstackedEtcd(c *Config) error {
+	if c.Cluster.Spec.ExternalEtcdConfiguration == nil || c.Cluster.Spec.ExternalEtcdConfiguration.MachineGroupRef == nil {
+		return nil
+	}
+
+	mc := c.SnowMachineConfig(c.Cluster.Spec.ExternalEtcdConfiguration.MachineGroupRef.Name)
+	if mc == nil {
+		return nil
+	}
+
+	for _, dni := range mc.Spec.Network.DirectNetworkInterfaces {
+		if dni.DHCP || dni.IPPoolRef == nil {
```

Comment:
Should these be different checks? The second one seems like it might just be missing and not really configured to DHCP.